draft-packages/stories/ZenNavigationBar.stories.tsx

```diff
@@ -3,15 +3,9 @@ import * as React from "react"
 import { Button, Icon } from "@kaizen/component-library"
 import { Link, Menu, ZenNavigationBar } from "@kaizen/draft-zen-navigation-bar"
 import { ColorScheme } from "@kaizen/draft-zen-navigation-bar/KaizenDraft/ZenNavigationBar/types"
-
-const caIcon = require("@kaizen/component-library/icons/ca-monogram.icon.svg")
-  .default
-const academyIcon = require("@kaizen/component-library/icons/academy.icon.svg")
-  .default
-const caMonogramIcon = require("@kaizen/component-library/icons/ca-monogram.icon.svg")
-  .default
-const supportIcon = require("@kaizen/component-library/icons/support.icon.svg")
-  .default
+import caIcon from "@kaizen/component-library/icons/ca-monogram.icon.svg"
+import academyIcon from "@kaizen/component-library/icons/academy.icon.svg"
+import supportIcon from "@kaizen/component-library/icons/support.icon.svg"
```

Comment:
For some reason, `ca-monogram.icon.svg` was imported twice, once as `caMonogramIcon` and again as `caIcon`. I removed the duplicate and changed everything to use `caIcon`